packages/recomponents/src/components/r-icon-button/r-icon-button.vue

```diff
@@ -6,7 +6,9 @@
         :disabled="disabled"
         v-tooltip="{text: tooltip}">
         <slot name="left-icon"></slot>
-        <slot/>
+        <div class="capitalize-the-first-letter">
```

Comment:
same here, can we add this to the parent class instead of changing the compontent's HTML structure?